agent/fw_laravel.c

```diff
@@ -1193,6 +1193,21 @@ static nr_status_t nr_laravel_should_assign_generic_path(const nrtxn_t* txn,
   return NR_FAILURE;
 }
 
+/*
+ * Ensure that the transaction is properly stopped when Laravel Horizon is being
+ * used by wrapping the handle method of the HorizonCommand and
+ * SupervisorCommand class.
+ */
+NR_PHP_WRAPPER(nr_laravel_horizon_end_txn) {
+  NR_UNUSED_SPECIALFN;
+  (void)wraprec;
+
+  nr_php_txn_end(1, 0 TSRMLS_CC);
+
+  nrl_verbosedebug(NRL_TXN, "Ending Laravel Horizon Transaction");
```

Comment:
The code is not very consistent about using logging subsystems, but most log messages in `fw_*.c` modules use `NRL_FRAMEWORK` logging subsystem. Since this code runs in the context of framework's instrumentation, I'd suggest using either `NRL_FRAMEWORK` or `NRL_INSTRUMENT` subsystem.

Reply:
Good point, changed: https://github.com/newrelic/newrelic-php-agent/pull/1123/commits/819903e18226a76fe8f0f6c626e875e2f2391d45